src/subdomains/generic/kyc/dto/sum-sub.dto.ts

```diff
@@ -293,30 +293,28 @@ const SumSubReasonMap: Record<SumSubRejectionLabels, string> = {
 };
 
 export function getSumsubResult(dto: SumSubWebhookResult): IdentShortResult {
-  if (dto.reviewStatus === ReviewStatus.INIT) IdentShortResult.PENDING;
   switch (dto.type) {
     case SumSubWebhookType.APPLICANT_PENDING:
-      return IdentShortResult.REVIEW;
+    case SumSubWebhookType.APPLICANT_REVIEWED: {
+      switch (dto.reviewStatus) {
+        case ReviewStatus.INIT:
+          return IdentShortResult.PENDING;
 
-    case SumSubWebhookType.APPLICANT_REVIEWED:
-      return dto.reviewResult.reviewAnswer === ReviewAnswer.GREEN ? IdentShortResult.SUCCESS : IdentShortResult.FAIL;
+        case ReviewStatus.PENDING:
+          if (dto.levelName === SumSubLevelName.CH_STANDARD) return IdentShortResult.REVIEW;
+          break;
 
-    case SumSubWebhookType.VIDEO_IDENT_STATUS_CHANGED:
-      if (dto.videoIdentReviewStatus === ReviewStatus.INIT) {
-        return IdentShortResult.PENDING;
-      }
-
-      if (dto.videoIdentReviewStatus === ReviewStatus.PENDING) {
-        return IdentShortResult.REVIEW;
-      }
-
-      if (dto.videoIdentReviewStatus === ReviewStatus.COMPLETED) {
-        return dto.reviewResult.reviewAnswer === ReviewAnswer.GREEN ? IdentShortResult.SUCCESS : IdentShortResult.FAIL;
+        case ReviewStatus.COMPLETED:
+          return dto.reviewResult.reviewAnswer === ReviewAnswer.GREEN
+            ? IdentShortResult.SUCCESS
+            : IdentShortResult.FAIL;
       }
       break;
+    }
 
-    default:
-      throw new BadRequest(`Unknown webhook type: ${dto.type}`);
```

Comment:
default is now missing. Is that on purpose? Would say default with exception makes sense or?

Reply:
I left it out on purpose because we now basically understand the process of the webhooks and don't need the other webhooks at all.